### PR TITLE
refactor: cleaning code

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/common/components/ui/Notification/index.tsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ const Notification: React.FC = () => {
       }}
     >
       {
+        // eslint-disable-next-line security/detect-object-injection
```
Comment (mmiranda-stackit) [inline src/common/components/ui/Notification/index.tsx:38]: Please avoid using inline eslint rules, it is better to use global rules, and if there is an exception, we should refactor whenever possible, or, as a last resource, create a new global rule disabling the entire rule.

File: src/features/auth/services/interfaces/auth-service.interface.ts [hunk 2]
```diff
@@ -26,5 +26,6 @@ export default interface IAuthService {
     passphrase: string,
     password: string,
     blockchainAccountAddress: string
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (mmiranda-stackit) [inline src/features/auth/services/interfaces/auth-service.interface.ts:29]: I think that we can type this response

File: src/features/auth/services/metamask.service.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import BusinessError from '@metis/common/exceptions/business-error';
 import IMetaMaskService from './interfaces/metamask-service.interface';
 
 export default class MetaMaskService implements IMetaMaskService {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (mmiranda-stackit) [inline src/features/auth/services/metamask.service.ts:8]: It is worth to investigate if there is a type definition for ethereum object, so we don't use any type.

**Review by mmiranda-stackit — CHANGES_REQUESTED** [review]
Please avoid using inline eslint rules disablers, please read my comments

**Comment by AlefrankM** [comment]
This changes were to resolve some conflicts. Now I'm gonna refactor it

File: src/features/auth/services/interfaces/auth-service.interface.ts (hunk 2 above)
Comment (AlefrankM) [reply src/features/auth/services/interfaces/auth-service.interface.ts:29]: I'm searching

**Review by mmiranda-stackit — DISMISSED** [review]
Looks good

**Review by mmiranda-stackit — APPROVED** [review]

**Pull request merged**